cohorts/functions.py

```diff
@@ -214,6 +221,44 @@ def count_filter_fn(filterable_effect, **kwargs):
     filterable_effect_function=lambda filterable_effect: (
         isinstance(filterable_effect.effect, FrameShift)))
 
+frameshift_indel_count = count_effects_function_builder(
+    "frameshift_indel_count",
+    only_nonsynonymous=False,
+    filterable_effect_function=lambda filterable_effect: (
+        filterable_effect.variant.is_indel and
+        isinstance(filterable_effect.effect, FrameShift)))
+
+exonic_frameshift_indel_count = count_effects_function_builder(
+    "exonic_frameshift_indel_count",
+    only_nonsynonymous=False,
+    filterable_effect_function=lambda filterable_effect: (
+        filterable_effect.variant.is_indel and
+        isinstance(filterable_effect.effect, FrameShift) and
+        isinstance(filterable_effect.effect, Exonic)))
+
+exonic_frameshift_deletion_count = count_effects_function_builder(
+    "exonic_frameshift_deletion_count",
+    only_nonsynonymous=False,
+    filterable_effect_function=lambda filterable_effect: (
+        filterable_effect.variant.is_deletion and
+        isinstance(filterable_effect.effect, FrameShift) and
+        isinstance(filterable_effect.effect, Exonic)))
+
+exonic_frameshift_insertion_count = count_effects_function_builder(
+    "exonic_frameshift_insertion_count",
+    only_nonsynonymous=False,
+    filterable_effect_function=lambda filterable_effect: (
+        filterable_effect.variant.is_insertion and
+        isinstance(filterable_effect.effect, FrameShift) and
+        isinstance(filterable_effect.effect, Exonic)))
+
+exonic_frameshift_snv_count = count_effects_function_builder(
```

Comment:
It looks like in varcode only `FrameshiftTruncation` effects inherit from `Frameshift`. AlternateStartCodon, PrematureStop & Startloss are all considered versions where we can either not predict the coding sequence (ie startloss) or they are considered in-frame.